ibis/pandas/execution.py

```diff
@@ -239,6 +245,17 @@ def execute_any_all_series(op, data, scope=None):
     return getattr(data, type(op).__name__.lower())()
 
 
+@execute_node.register(ops.CountDistinct, pd.Series)
+def execute_count_distinct_series(op, data, scope=None):
+    # TODO(phillipc): Does count distinct have a mask?
+    return data.nunique()
+
+
+@execute_node.register(ops.Count, pd.DataFrame, type(None))
+def execute_count_frame(op, data, _, scope=None):
+    return len(data)
```

Comment:
No, a `Count` operation on table means "count the number of rows in the table": https://github.com/ibis-project/ibis/blob/master/ibis/expr/api.py#L1854-L1862

We could think about defining the operation you're talking about as something else that's more general like "apply reduction `f` to every column", which would return a thing that looks like a table with one row.